.kokoro/dashboard.sh

```diff
@@ -20,9 +20,38 @@ scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
 ## cd to the parent directory, i.e. the root of the git repo
 cd ${scriptDir}/..
 
+output="$scriptDir/dashboard/target/tmp/output.txt"
 ## Move into the dashboard directory
 cd dashboard/
 
 echo -e "\n******************** BUILDING THE DASHBOARD ********************"
 
 mvn --fail-at-end clean install
+INSTALL_RETURN_CODE=$?
+RETURN_CODE=${INSTALL_RETURN_CODE}
+
+LINE_COUNT=0
+
+case ${JOB_TYPE} in
+dependency-convergence-check)
+    mvn exec:java -Dexec.args="-f ../pom.xml --report"
```

Comment:
I suggest adding a CLI parameter for `--output-file=path/to/file` which would tell the plugin where to put the output file rather than hard-coding it.